templates/referent/municipal/running_mate/detail.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends 'referent/_layout.html.twig' %}
+
+{% block referent_content %}
+    <a href="{{ path('app_referent_municipal_running_mate_request') }}" class="back-to-list icon--with-text b__nudge--top-15 b__nudge--bottom">
+        {{ include('components/caret--left.html.twig') }}
+        Retour à la liste des candidats colistiers
+    </a>
+
+    <hr />
+
+    <div class="l__wrapper--slim">
+        <strong>Nom :</strong> {{ runningMateRequest.lastName }}<br>
+        <strong>Prénom :</strong> {{ runningMateRequest.firstName }}<br>
+        <strong>Téléphone :</strong> {{ runningMateRequest.phone ? runningMateRequest.phone | phone_number_format : '' }}<br>
```

Comment:
Can you remove the whitespaces on the twig filters please ?

`runningMateRequest.phone | phone_number_format`

 =>

`runningMateRequest.phone|phone_number_format`